lib/logstash/devutils/rspec/shared_examples.rb

```diff
@@ -0,0 +1,19 @@
+require 'rspec/wait'
+
+RSpec.shared_examples "an interruptible input plugin" do
+  describe "stop" do
```

Comment:
it's common ruby idiom to use `#foo` for instance method and `.foo` for class methods.

But I would probably use this for the description of this shared examples..
```ruby
context "when stopping a plugin" do
end
```